MapboxNavigation/DayStyle.swift

```diff
@@ -45,13 +45,12 @@ private extension UIFont {
  */
 @objc(MBDayStyle)
 open class DayStyle: Style {
-    public required init() {
-        super.init()
-        mapStyleURL = MLNStyle.navigationGuidanceDayStyleURL
+    @objc public required init(mapStyleURL: URL) {
+        super.init(mapStyleURL: mapStyleURL)
         styleType = .day
         statusBarStyle = .default
     }
-    
```

Comment:
I am wondering about this change. Don't we have auto formatting enabled that should make sure we don't have these kinds of changes? Did you enable the auto formatting? See: https://github.com/maplibre/maplibre-navigation-ios/blob/main/CONTRIBUTING.md#enable-automatic-code-formatting

Reply:
BTW: I don't really mind this particular change, I just expected that these types of things don't happen anymore.